scripts/tsgen/bin/Parser.js

```diff
@@ -12,7 +12,7 @@ class Parser {
         this.resolveObjects(docs);
         // removes members inherited from classes
         // possibly could be avoided if mixins were defined as such before JSDoc parses them and then we could globally remove all inherited (not
-        // overriden) members globally from the parsed DB
+        // overridden) members globally from the parsed DB
```

Comment:
It's possible this is a generated file, if this or any other file is generated/from an upstream, please let me know. I've tried to skip such files (and when I can find an upstream, I'm happy to pass fixes on to them as well).

Reply:
This file is auto-generated and should not be edited in a PR, thanks.